src/Accelerators/NNPA/Conversion/ZLowToLLVM/ZLowToLLVM.cpp

```diff
@@ -1542,10 +1542,11 @@ class ZLowDLF16ToF32Lowering : public ConvertToLLVMPattern {
         Type vecTypeI32 = LLVM::getFixedVectorType(i32Ty, 4);
         Type vecTypeF32 = LLVM::getFixedVectorType(f32Ty, 4);
 
-        // SIMD instruction in string for z/Linux.
+        // SIMD instruction in string for z/Linux and z/OS.
         // Convert and lengthen from DLF16: VCLFN(H/L) V1,V2,M3,M4
         // M3 = 2 = FP32, M4 = 0 = DLF16
-        const char *asmStr = "VCLFNH $0,$2,2,0 \n\t VCLFNL $1,$2,2,0 \n\t";
+        const char *asmStr = "       VCLFNH $0,$2,2,0       \n\t"
```

Comment:
@mikeessen is it important to have the spaces for zOS compatibility? If so, do you mind stating this in a comment above, so that next time we add an asm statement, we know we have to do so (or the person fixing it for zOS can find a trace of what to do). Thanks

Reply:
Yes, the spaces are required for the z/OS assembler.  Thanks for the comments and approving!

Reply:
Thanks for adding the comments, it will be useful for further updates.